be/src/olap/data_dir.h

```diff
@@ -93,6 +94,14 @@ class DataDir {
     // load data from meta and data files
     OLAPStatus load();
 
+    void add_pending_ids(const std::string& id);
+
+    void remove_pending_ids(const std::string& id);
+
+    void perform_path_gc();
```

Comment:
You should add comment about these two functions, and their relations

Reply:
ok